samples/azure-service-bus-netstandard/native-integration/sample.md

```diff
@@ -46,7 +46,7 @@ For a native message to be processed, NServiceBus endpoints using the Azure Serv
 
 snippet: NecessaryHeaders
 
-NOTE: The `NServiceBus.EnclosedMessageTypes` property must contain the the fully-qualified name of the type expected by the NServiceBus endpoint.
+NOTE: The [`NServiceBus.EnclosedMessageTypes` header property](/nservicebus/messaging/headers.md#serialization-headers-nservicebus-enclosedmessagetypes) must contain the type FullName (namespace and name) expected by the NServiceBus endpoint. Not to be confused with [Assembly Qualified Name](https://docs.microsoft.com/en-us/dotnet/api/system.type.assemblyqualifiedname) which also includes the assembly and version of the type.
```

Comment:
It can contain both the FullName and the assembly qualified name, we handle both. When we send a message we ([IMO incorrectly](https://github.com/Particular/DeveloperExperience/issues/269)) use the Asm qualified name.

https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Pipeline/Outgoing/SerializeMessageConnector.cs#L58

Does this make sense? (Then this need to be reworded)

Reply:
@andreasohlund ok, please review again. I reworded it that it atleast must contain the FullName, but that is can contain the Assembly Qualified Name.

Keep in mind that this page is about creating a native integration where a sender must prep the header values to be compatible with NServiceBus.